server/scriptController.js

```diff
@@ -20,7 +22,102 @@ const scriptController = {
       .catch((error) => {
         return next({
           log: `error: ${error} occured when getting script titles`,
-          message: 'error in getScriptTitles in scriptController' ,
+          message: 'error in getScriptTitles in scriptController',
+        });
+      });
+  },
+  saveScript: (req, res, next) => {
+    const MAXFILESIZE = 50 * 1024 * 1024; //50MB
+
+    // configure multer storage to upload files to the path from the .env file
+    const storage = multer.diskStorage({
+      destination: function (req, file, cb) {
+        cb(null, process.env.UPLOADPATH);
+      },
+      filename: function (req, file, cb) {
+        cb(null, Date.now() + file.originalname);
```

Comment:
We probably need to validate the originalname so that it doesn't contain any harmful characters and is of a reasonable length.